front/lib/production_checks/checks/scrub_deleted_core_document_versions.ts

```diff
@@ -71,6 +72,7 @@ export const scrubDeletedCoreDocumentVersionsCheck: CheckFunction = async (
             storage,
             data_source: d.data_source,
             document_id: d.document_id,
+            timestamp: d.timestamp,
```

Comment:
I thought that `timestamp` was the timestamp from the external resource. Internally, we use `created`.